optaplanner-core/src/test/java/org/optaplanner/core/impl/testdata/domain/reflect/field/TestdataFieldAnnotatedSolution.java

```diff
@@ -75,6 +74,7 @@ public void setScore(SimpleScore score) {
     // Complex methods
     // ************************************************************************
 
+    @PlanningFactCollectionProperty
```

Comment:
For TestdataFieldAnnotatedSolution, I 'd annotated the @PlanningFactCollectionProperty on the fields.